src/mumble/AudioInput.cpp

```diff
@@ -6,6 +6,7 @@
 #include "AudioInput.h"
 
 #include "AudioOutput.h"
+#include "AudioOutputSample.h"
```

Comment:
What is this needed for? :eyes:

Reply:
Removed. I had an incomplete type in an intermediate state where I tried to use slots.